contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1049,6 +1050,15 @@ define_custom_variables(void)
 				 GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE,
 				 NULL, NULL, NULL);
 
+	DefineCustomIntVariable("babelfishpg_tsql.dump_restore_min_oid",
+				 gettext_noop("All new OIDs should be greater than this number during dump and restore"),
+				 NULL,
+				 &babelfish_dump_restore_min_oid,
+				 FirstNormalObjectId, 0, INT_MAX,
```

Comment:
is INT_MAX for unsigned or signed? we should make sure it matches max oid value

Reply:
Good catch! Will change it to UINT_MAX.